x-pack/plugins/apm/public/plugin.ts

```diff
@@ -162,7 +162,24 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
       order: 8500,
       euiIconType: 'logoObservability',
       category: DEFAULT_APP_CATEGORIES.observability,
-
+      meta: {
+        keywords: [
+          'RUM',
+          'Real User Monitoring',
+          'DEM',
+          'Digital Experience Monitoring',
+          'EUM',
+          'End User Monitoring',
+          'UX',
+          'Javascript',
+          'APM',
```

Comment:
Is APM team ok with this?

Reply:
Seeing as the first result in the list will always be the APM app, I think we're OK with additionally showing a link to the UX app as well as a secondary option.

<img width="620" alt="Screenshot 2021-02-09 at 09 56 19" src="https://user-images.githubusercontent.com/4104278/107339611-29591800-6abd-11eb-9d6d-957ffbf37913.png">